### DISCUSSION TITLE
Extra scrollable space after last line

### DISCUSSION THREAD

**Discussion by farginnoob** [TriliumNext/Trilium#4136] [Ideas]
In things like VSCode, there's an option (Scroll Beyond Last Line) you can toggle that will let you scroll beyond the last line in the document, which is nice if you don't want to be confined to the very last line in a note in the editor.

In the VSCode implementation, you can scroll until only the last line of the document is visible. I'm not sure what the implementation details are, but it feels very nice to be able to scroll a document up so that the place I'm currently writing at is at eye level.

**Comment by FliegendeWurst** [comment]
This was previously discussed in #2728.
